### PR TITLE
move some constants to rails environment

### PR DESCRIPTION
**Pull request by ashkitten**
figured i'd hand this up

### CODE REVIEW

**Comment by ThibG** [comment]
Some of these values are also checked client-side in some JS… I'll have to think about how to get those values there (I don't think the `initialState` thing is used in the settings pages)

**Comment by ThibG** [comment]
Upstream has stopped using those hardcoded values in the javascript bits, so I'll probably merge this PR once I'm done merging upstream!

File: app/views/settings/profiles/show.html.haml [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
   .fields-row
     .fields-row__column.fields-group.fields-row__column-6
-      = f.input :display_name, wrapper: :with_label, hint: t('simple_form.hints.defaults.display_name', count: 30 - @account.display_name.size).html_safe
-      = f.input :note, wrapper: :with_label, hint: t('simple_form.hints.defaults.note', count: 500 - @account.note.size).html_safe
+      = f.input :display_name, wrapper: :with_label, hint: t('simple_form.hints.defaults.display_name', count: (ENV['MAX_DISPLAY_NAME_CHARS'] || 30).to_i - @account.display_name.size).html_safe
+      = f.input :note, wrapper: :with_label, hint: t('simple_form.hints.defaults.note', count: (ENV['MAX_BIO_CHARS'] || 30).to_i - @account.note.size).html_safe
```
Comment (ThibG) [inline app/views/settings/profiles/show.html.haml]: This has changed since last upstream change. Can you rebase?

**Comment by ashkitten** [comment]
should be fine now, i also realized i can simplify it a bit

**Comment by ashkitten** [comment]
oh, is github broken again, that's fun :D

**Pull request merged**